<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditApp</class>
 <widget class="QDialog" name="EditApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>315</y>
     <width>561</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="AppForm">
    <item row="0" column="1">
     <widget class="QTextEdit" name="newName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="YearBuilt">
      <property name="text">
       <string>Year Built : </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="bashPath"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="newDescription"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="PathToBash">
      <property name="text">
       <string>Path To Bash : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="author"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Chef">
      <property name="text">
       <string>Chef : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="newYear"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../waiter.gif</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="Done">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
